src/params/constants.ts

```diff
@@ -7,7 +7,7 @@ export const units: Unit[] = [
   { name: 'nano', value: 1e-9 },
   { name: 'micro', value: 1e-6 },
   { name: 'mili', value: 1e-3 },
-  { name: '-', value: 1 },
+  { name: 'KSM', value: 1 },
```

Comment:
This is an unnecessary change

Reply:
yes, we will be working on other parachains apart from Kusama, so probably we don't want to hard code it

Reply:
Done it.